_episodes/03-packaging-installing.md

```diff
@@ -28,12 +28,73 @@ Pip is used to install packages from FIXME.
 
 ## Python Packages
 
-A module is an installable unit of code.
+A module is a piece of code that serves a paticular functionalty. In python moduls ecd with a **.py** extension and the name of the module is the name of the file. A module can contain classes, functions or a combination of both. Packages are namespaces which contain multiple packages and modules.
 
 
-The file `__init__.py`
+Working with Python packages and modules is simple. We need to:
 
-FIXME: describe structure of how a package works in python
+* Create a directory and give it package's name.
+* Put modules in it.
+* Create a __init__.py file in the directory
+
+The __init__.py file tells python that the directory is supposed to be read as a package.
+
+e.g.
+
+Let us create a package called **Vehicles** with two modules **Land** and **Water**.
+
+### Step 1: Creating a directory
+Create a directory called **Vehicles**
+
+### Step 2: Adding Modules
+
+```
+class Land:
+    def __init__(self):
```

Comment:
can the example be functions instead of classes they're used more by novice programmers and are included in the material that is listed as prerequisite, but classes are not.